packages/common/pipes/file/parse-file.pipe.ts

```diff
@@ -25,13 +27,20 @@ export class ParseFilePipe implements PipeTransform<any> {
       exceptionFactory,
       errorHttpStatusCode = HttpStatus.BAD_REQUEST,
       validators = [],
+      fileIsRequired = true,
     } = options;
 
     this.exceptionFactory =
       exceptionFactory ||
       (error => new HttpErrorByCode[errorHttpStatusCode](error));
 
     this.validators = validators;
+
+    this.validators.unshift(
+      new RequiredFileValidator({
+        required: fileIsRequired,
+      }),
```

Comment:
isn't this redundant? not sure if I follow

```suggestion
      new RequiredFileValidator(),
```